forte/data/ontology/configs/stanfordnlp_ontology.json

```diff
@@ -1,58 +1,62 @@
 {
-  "ontology_name": "forte.data.ontology.stanfordnlp_ontology",
-  "imports": [
-    "forte.data.ontology.top",
-    "forte.data.ontology.base_ontology"
-  ],
-  "entry_definitions": [
+  "name": "stanfordnlp_ontology",
+  "definitions": [
     {
-      "entry_name": "forte.data.ontology.stanfordnlp_ontology.Token",
+      "entry_name": "ft.onto.stanfordnlp_ontology.Token",
       "parent_entry": "forte.data.ontology.top.Annotation",
       "attributes": [
         {
-          "attribute_name": "lemma",
-          "attribute_type": "str"
+          "name": "lemma",
+          "type": "str"
         },
         {
-          "attribute_name": "pos_tag",
-          "attribute_type": "str"
+          "name": "pos_tag",
+          "type": "str"
         },
         {
-          "attribute_name": "upos",
-          "attribute_type": "str"
+          "name": "upos",
+          "type": "str"
         },
         {
-          "attribute_name": "xpos",
-          "attribute_type": "str"
+          "name": "xpos",
+          "type": "str"
         }
       ]
     },
     {
-      "entry_name": "forte.data.ontology.stanfordnlp_ontology.Sentence",
+      "entry_name": "ft.onto.stanfordnlp_ontology.Sentence",
       "parent_entry": "forte.data.ontology.top.Annotation",
       "attributes": [
         {
-          "attribute_name": "tokens",
-          "attribute_type": "List",
-          "element_type": "forte.data.ontology.stanfordnlp_ontology.Token"
+          "name": "tokens",
+          "type": "List",
+          "item_type": "ft.onto.stanfordnlp_ontology.Token"
         }
       ]
     },
     {
-      "entry_name": "forte.data.ontology.stanfordnlp_ontology.Document",
+      "entry_name": "ft.onto.stanfordnlp_ontology.Document",
       "parent_entry": "forte.data.ontology.top.Annotation"
     },
     {
-      "entry_name": "forte.data.ontology.stanfordnlp_ontology.Dependency",
+      "entry_name": "ft.onto.stanfordnlp_ontology.Dependency",
       "parent_entry": "forte.data.ontology.top.Link",
-      "parent_type": "forte.data.ontology.stanfordnlp_ontology.Token",
-      "child_type": "forte.data.ontology.stanfordnlp_ontology.Token",
+      "class_attributes": [
```

Comment:
I think we don't need to specify the ``class_attributes`` here, just specify the parent type and child type.